File: challenge1/staircase.py
def staircase(n: int):
    # Write your code here
    hash = n
    rta = ""
    for i in range(n):
        x = [" "] * n
        x[hash-1:n] = ["#"] * (n - hash + 1)
        rta += "".join(x) + "\n"
        hash -= 1
    print(rta)

if __name__ == "__main__":
    staircase(7)

File: challenge1/staircase.js
function staircase(n) {
  // Write your code here
  let hash = n;
  let rta = "";
  for (let i = 0; i < n; i++) {
    let x = Array(n).fill(" ");
    let arr2 = x.fill("#", hash - 1, n);
    arr2 = arr2.join("");
    hash--;
    rta += arr2.toString() + "\n";
  }
  console.log(rta);
}

staircase(7);
